src/discussions/comments/comment/CommentHeader.jsx

```diff
@@ -24,6 +27,21 @@ function CommentHeader({
   const colorClass = AvatarOutlineAndLabelColors[comment.authorLabel];
   const hasAnyAlert = useAlertBannerVisible(comment);
 
+  const actions = useActions({
+    ...comment,
+    postType,
+  });
+  const actionIcons = actions.find(({ action }) => action === 'endorsed');
```

Comment:
We have multiple actions with the name `endorsed` in the actions list. Find will get the first action. If you need all the action with `endorsed` then you need to change the array function `find` to `filter`.  
Please update the `action === 'endorsed'`  hard-coded string comparison with the already defined constant for endorsed.